B827EBFFFE00E326.json

```
@@ -2,9 +2,9 @@
 	"gateway_conf": {
 	 	"gateway_ID": "B827EBFFFE00E326",
 		"servers": [ 
-			{ "server_address": "router.eu.thethings.network",
-				"serv_port_up": 1700,
-				"serv_port_down": 1700,
+			{ "server_address": 127.0.0.1,
```

You're missing the quotes around the ip:
```suggestion
			{ "server_address": "127.0.0.1",
```